tic-tac-toe/script.js

```diff
@@ -0,0 +1,157 @@
+	buttonOne.removeEventListener("click", click);
+	});
+
+console.log("You clicked");
+
+buttonTwo.addEventListener("click", function(click) {
+//	for (var i = 0; i < buttons.length; i++) 
+	counter	++;
+	click.preventDefault();
+		if (counter % 2 == 0) {
+			(buttonTwo.innerHTML = "O", buttonTwo.style.background = "palevioletred");
+		} else {(buttonTwo.innerHTML = "X", buttonTwo.style.background = "lightblue")};
+
+	});
+ // var clickHandler = function() {
+ //        this.removeEventListener('click', clickHandler); }
+
+buttonThree.addEventListener("click", function(click) {
+//	for (var i = 0; i < buttons.length; i++) 
+		counter	++;
+		click.preventDefault();
+		if (counter % 2 == 0) {
+		(buttonThree.innerHTML = "O", buttonThree.style.background = "palevioletred");
+		} else {(buttonThree.innerHTML = "X", buttonThree.style.background = "lightblue")};
+		
+	});
+
+buttonFour.addEventListener("click", function(click) {
+//	for (var i = 0; i < buttons.length; i++) 
+		counter	++;
+		click.preventDefault();
+		if (counter % 2 == 0) {
+			(buttonFour.innerHTML = "O", buttonFour.style.background = "palevioletred");
+		} else {(buttonFour.innerHTML = "X", buttonFour.style.background = "lightblue")};
+		
+	});
+
+buttonFive.addEventListener("click", function(click) {
+//	for (var i = 0; i < buttons.length; i++) 
+		counter	++;
+		click.preventDefault();
+		if (counter % 2 == 0) {
+			(buttonFive.innerHTML = "O", buttonFive.style.background = "palevioletred");
+		} else {(buttonFive.innerHTML = "X", buttonFive.style.background = "lightblue")};
+		
+	});
+
+buttonSix.addEventListener("click", function(click) {
+//	for (var i = 0; i < buttons.length; i++) 
+		counter	++;
+		click.preventDefault();
+		if (counter % 2 == 0) {
+	(buttonSix.innerHTML = "O", buttonSix.style.background = "palevioletred");
+		} else {(buttonSix.innerHTML = "X", buttonSix.style.background = "lightblue")};
+		
+	});
+
+buttonSeven.addEventListener("click", function(click) {
+//	for (var i = 0; i < buttons.length; i++) 
+		counter	++;
+		click.preventDefault();
+		if (counter % 2 == 0) {
+		(buttonSeven.innerHTML = "O", buttonSeven.style.background = "palevioletred");
+		} else {(buttonSeven.innerHTML = "X", buttonSeven.style.background = "lightblue")};
+		
+	});
+buttonEight.addEventListener("click", function(click) {
+//	for (var i = 0; i < buttons.length; i++) 
+		counter	++;
+		click.preventDefault();
+		if (counter % 2 == 0) {
+			(buttonEight.innerHTML = "O", buttonEight.style.background = "palevioletred");
+		} else {(buttonEight.innerHTML = "X", buttonEight.style.background = "lightblue")};
+		
+	});
+buttonNine.addEventListener("click", function(click) {
+//	for (var i = 0; i < buttons.length; i++) 
+		counter	++;
+		click.preventDefault();
+		if (counter % 2 == 0) {
+			(buttonNine.innerHTML = "O", buttonNine.style.background = "palevioletred");
+		} else {(buttonNine.innerHTML = "X", buttonNine.style.background = "lightblue")};
+	});
+
```

Comment:
Glad to see you got it working! It works fine as is, but you can try to DRY up this code by creating a more generic function. To make your code more concise, try making a generic function that you pass to the eventListener that takes the button in as a parameter and changes it so that you only need to run it once.